<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SidePanel</class>
 <widget class="QWidget" name="SidePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SidePanel</string>
  </property>
  <property name="styleSheet">
   <string notr="true">        
            QWidget {
                background-color: #12283A; 
                color: #FFFFFF;
                font-family: &quot;Microsoft YaHei&quot;;
            }

            QLabel#titleLabel {
                font-size: 14px;
                font-weight: bold;
                padding: 8px 10px;
                background-color: #0A1C2B;
                border-bottom: 1px solid #1F3C52;
            }

            QToolButton {
                 background-color: transparent;
                border: none;
                color: #FFFFFF;
                font-size: 12px;
                padding: 12px 0;
                text-align: center;
                width: 60px;
            }
            QToolButton:hover {
                background-color: #1F3C52;
            }
            QToolButton:pressed {
                background-color: #162E40;
            }

            QGroupBox {
                border: 1px solid #1F3C52;
                border-radius: 4px;
               margin: 10px;
                padding: 10px;
            }
            QGroupBox::title {
                color: #B0C8D8;
                font-size: 12px;
               subcontrol-origin: margin;
                left: 10px;
                padding: 0 5px;
            }

            QMenu {
                background-color: #1F3C52;
                border: 1px solid #162E40;
                color: #FFFFFF;
                font-size: 12px;
            }
            QMenu::item {
                padding: 6px 20px;
                background-color: transparent;
            }
            QMenu::item:selected {
                 background-color: #162E40;
            }

             QListWidget {
                 border: 1px solid #1F3C52;
                border-radius: 4px;
                background-color: #12283A;
                color: #FFFFFF;
                 font-size: 12px;
                padding: 5px;
            }
            QListWidget::item {
                padding: 6px;
                border-bottom: 1px solid #1F3C52;
            }
            QListWidget::item:selected {
                 background-color: #1F3C52;
            }
         </string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;测试123&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="icon">
       <iconset>
        <selectedon>:/img/SidePanel/recover.png</selectedon>
       </iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>316</width>
     <height>853</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideLeft</enum>
   </property>
   <column>
    <property name="text">
     <string>卫星(708)</string>
    </property>
   </column>
   <item>
    <property name="text">
     <string>通信卫星</string>
    </property>
    <item>
     <property name="text">
      <string>中星-9</string>
     </property>
     <item>
      <property name="text">
       <string>指向</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>属性</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>通信 设置</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>导航设置</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>关联站点</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>删除</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>添加星座</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>中星-6E</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中星-6D</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中星-6C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中星-6B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中星-4A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中星-3A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中星-2E</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中星-2D</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中星-2C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中星-2A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中星-26</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中星-1E</string>
     </property>
    </item>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_red">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>红方</string>
   </property>
   <property name="text">
    <string>红方</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/img/SidePanel/scene2.png</normaloff>:/img/SidePanel/scene2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_blue">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>蓝方</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/img/SidePanel/scene2.png</normaloff>:/img/SidePanel/scene2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_demand">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>需求</string>
   </property>
   <property name="text">
    <string>需求</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/img/SidePanel/need.png</normaloff>:/img/SidePanel/need.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mark">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>标绘</string>
   </property>
   <property name="text">
    <string>标绘</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/img/SidePanel/marking1.png</normaloff>:/img/SidePanel/marking1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_user">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>用户</string>
   </property>
   <property name="text">
    <string>用户</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/img/SidePanel/role1.png</normaloff>:/img/SidePanel/role1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_scene">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>场景</string>
   </property>
   <property name="text">
    <string>场景</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/img/SidePanel/scene2.png</normaloff>:/img/SidePanel/scene2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="GRID/paceGrid/resources/qrc/SidePanel.qrc"/>
 </resources>
 <connections/>
</ui>
